pwnlib/context/__init__.py

```diff
@@ -1071,6 +1072,10 @@ def log_console(self, stream):
             stream = open(stream, 'wt')
         return stream
 
+    @_validator
+    def local_libcdb(self, path):
```

Comment:
Can you add a docstring describing what this is and how to use it please? And maybe mention the default value if it doesn't show up automatically. A link to the libc-database repo can be useful too so you know where to get the scripts for your own libc database.